### ISSUE TITLE
The hub site association is there when you unregister the hub site and delete the hub site.

### ISSUE THREAD

**Issue by longnlp** [MicrosoftDocs/OfficeDocs-SharePoint#3113]
Hello

This document mentioned that "f you're a global or SharePoint admin in Microsoft 365, you can make a hub site no longer a hub site (unregister it as a hub site). Make sure you do this before you delete the hub site. When you unregister a hub site, any sites still associated with the hub will be disassociated the next time a user accesses them"

And I made the following test:
1. Unregister the hub site
2. Delete the hub site
3. Access the site which is associated to hub site, and try to change the theme, and it says "Your site is connected to xxx hub site and is set to automatically adopt the same theme"

So my question is: how long for SharePoint to disconnect the associated when the hub site is unregistered and deleted?

Thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 96fc2661-679c-dff4-7e87-651f0b154513
* Version Independent ID: 9a0a4a29-c1e9-572a-9009-ed3c320a070d
* Content: [Remove a hub site - SharePoint in Microsoft 365](https://docs.microsoft.com/en-us/sharepoint/remove-hub-site#feedback)
* Content Source: [SharePoint/SharePointOnline/remove-hub-site.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/public/SharePoint/SharePointOnline/remove-hub-site.md)
* Service: **sharepoint-online**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

**Comment by get-itips** [comment]
Hello @longnlp I am looking into this...

**Comment by get-itips** [comment]
Hello @longnlp how are you?
I tested with 2 different sites associated with two different hub sites.
On one case, I am still waiting for the disassociaton to be effective to change the theme.
On the other case, I was able to speed up the disassociaton by editing the hub association and selecting "None".
When the first case finally disassociates, I'll review if an update to the article is neccesary. 
Thank you!
